ws-server/src/server.rs

```diff
@@ -95,8 +92,9 @@ impl Server {
 
 async fn background_task(
 	socket: tokio::net::TcpStream,
-	methods: Arc<Methods>,
 	conn_id: ConnectionId,
+	methods: Arc<Methods>,
+	cfg: Arc<Builder>,
```

Comment:
why is this an `Arc`? 

Builder should be able implement Copy because it's cheap but for future additions maybe?